raft/raftpb/raft.proto

```diff
@@ -43,6 +43,7 @@ enum MessageType {
 	MsgHeartbeat     = 8;
 	MsgHeartbeatResp = 9;
 	MsgUnreachable   = 10;
+	MsgSnapStatus    = 11;
```

Comment:
> If it's only sent on failure then

We still need to report the success. The actual MsgAppResp from the remote might lost. :(
That is why I want to attach CommitIndex the MsgHeartbeatResp.